agendas/2024/sprint-kickoff/YYYY-MM-DD-Flow-Sprint-Kickoff-Template.md

```diff
@@ -0,0 +1,219 @@
+# Team Wins 🎉
+
+- 
+
+### Mainnet Uptime - Last 14 days (DD/MM/YY to DD/MM/YY) - Vishal
+
+|                         | Target | Current Score | Error budget used |
+|:------------------------|:------:|:-------------:|:-----------------:|
+| Collection Finalization | 99.9%   |    100%       |       0%         |
+| Block Finalization      | 99.9%   |    100%       |       0%         |
+| Transaction Execution   | 99.9%   |    100%       |       0%         |
+| Block Sealing           | 99.9%   |    100%       |       0%         |
+| Access API Liveness     | 99.9%   |    100%       |       0%         |
+
+#### Incidents
+- 
+
+### **Cadence and Virtual Machine** Jan
+Objective: 
+
+**Done last sprint**
+
+- General description of completed items
+- List of Closed issues
+
+
+**This sprint**
+
+- General description of sprint goal
+- List of issues to be worked on
+
+
+**On Hold**
+
+- List of issues on hold
+
+
+**Active Epics**
+
+- Main epics that are being tackled, and currently active for this working group
+
+---
+
+### **Core Protocol** Jerome
+Objective: 
+
+**Done last sprint**
+
+- General description of completed items
+- List of Closed issues
+
+
+**This sprint**
+
+- General description of sprint goal
+- List of issues to be worked on
+
+
+**On Hold**
+
+- List of issues on hold
+
+
+**Active Epics**
+
+- Main epics that are being tackled, and currently active for this working group
+
+---
+
+### **DeFi** Jerome
+Objective: 
+
+**Done last sprint**
+
+- General description of completed items
+- List of Closed issues
+
+
+**This sprint**
+
+- General description of sprint goal
+- List of issues to be worked on
+
+
+**On Hold**
+
+- List of issues on hold
+
+
+**Active Epics**
+
+- Main epics that are being tackled, and currently active for this working group
+
+---
+
+### **User Experience** Greg
+Objective: 
```

Comment:
Will this change from Sprint to sprint? How does it differ from sprint goal?

Reply:
I meant this to be a place to be constantly reminded of the cycle objective. Let me update that.